.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,30 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  lint_python:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]    # [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.9.0-rc.1]  # [2.7, 3.5, 3.6, 3.7, 3.8, pypy3]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: pip install black codespell flake8 isort pytest
+      - run: black --check . || true
+      # - run: black --diff . || true
+      # - if: matrix.python-version >= 3.6
+      #  run: |
+      #    pip install black
+      #    black --check . || true
+      - run: codespell --quiet-level=2 --skip="*.json"  # --ignore-words-list=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: isort --profile black . || true
+      - run: pip install -r requirements.txt || true
+      - run: pytest . || true
```

Comment:
Oh, wow thanks for this!
What is the use of `isort`, `black` and `pytest` in here? Is it just an option if I want to incorporate it in the future? I'm asking because it will be `true` no matter what.

Reply:
Correct...  Tighten the screws by removing the trues.  Also removing the flake8 select option (after psf/black).

Reply:
I'm a little bit new to CI but thanks for doing this and also changing the typo. I'll merge this now. Also, I have to do a bit of research into CI now. 😁